web/template/layout/notification-banner.gohtml

```diff
@@ -1,18 +1,18 @@
 {{ define "notification-banner" }}
-    <div class="govuk-notification-banner {{ if .Success }}govuk-notification-banner--success{{ end }}" role="alert" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+    <div class="govuk-notification-banner {{ if .Success }}govuk-notification-banner--success{{ end }}" role="{{ if .Success }}alert{{ else }}region{{ end }}" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
```

Comment:
I noticed this in the [docs](https://design-system.service.gov.uk/components/notification-banner/)